frontend/src/components/individuals/IndividualsDetailContent.js

```diff
@@ -42,6 +42,7 @@ const IndividualsDetailContent = ({individualsByID, taxonsByID, get}) => {
             <Descriptions bordered={true} size="small" column={3}>
                 <Descriptions.Item label="ID">{individual.id}</Descriptions.Item>
                 <Descriptions.Item label="Name">{individual.name}</Descriptions.Item>
+                <Descriptions.Item label="Alias">{individual.alias}</Descriptions.Item>
```

Comment:
I would rearrange the fields to be in the following order to keep the father mother and pedigree together: 
ID  name alias
taxon sex cohort
pedigree mother father